CONTRIBUTING.md

```diff
@@ -1,8 +1,16 @@
 # Contributing to ClickHouse
 
+ClickHouse is an open project, and you can contribute to it in many ways. You can help with ideas, code, or documentation. We appreciate any efforts that help us to make the project better.
+
+Thank you :-)
+
 ## Technical info
-Developer guide for writing code for ClickHouse is published on official website alongside the usage and operations documentation:
-https://clickhouse.yandex/docs/en/development/architecture/
+
+You can find the "Contacts" section on the project's [website](https://clickhouse.yandex/). Select any way of communicating with us and discuss ideas.
```

Comment:
I'd remove this, it's not clear who is "us" and there are too many ways of communication